src/pages/staff/dashboard/NotificationsPage.jsx

```
@@ -54,26 +60,75 @@ export default function NotificationsPage() {
-          <EmptyState icon="🔔" headline="No system notifications." copy="We’ll post important system updates here." />
+        <div className="px-4 py-3 border-b border-gray-100 text-sm font-medium text-gray-700">Community activity</div>
+        {communityActivity.length === 0 ? (
+          <EmptyState icon="✅" headline="You're all caught up." copy="No new community updates." />
+        ) : (
+          <ul role="list" className="divide-y divide-gray-100">
+            {communityActivity.map((item) => (
+              <li key={item.id}>
+                <button
+                  type="button"
+                  onClick={() => {
+                    try { track('notification_open', { communityId: item.id }); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)